### ISSUE TITLE
KB based key value memory network

### ISSUE THREAD

**Issue by ghost** [jojonki/movie_dialog#1]
hi, I am trying to use kb for question answering. In your code I was able to load key-value pairs from movie KB as follows:
kv_pairs = load_kv_pairs(pathname, entities, 100, True)
but I couldn't find the implementation of vectorize_kv_pairs(kv_pairs, 10, 30, entities) function.
Thank you.

**Comment by samhavens** [comment]
@batselm in the notebook, about halfway down you see `from process_data import vectorize_kv`. So I looked in `process_data.py` and found it on [line 190](https://github.com/jojonki/key-value-memory-networks/blob/master/process_data.py#L190).

**Comment by Joseph1314** [comment]
hi,excuse me ,as it shows in the keras-memnn.ipynb, is the final accuracy on validation set about 23%？
